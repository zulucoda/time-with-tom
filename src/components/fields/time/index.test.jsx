import { render } from '@testing-library/react';
import {
  DateAndTimePickerProviderWrapperTestUtil,
  FormikWrapperTestUtil,
} from '../../../test-util';
import TimeField from '.';
import React from 'react';

describe('Components - Fields - Time - Unit Test', () => {
  test('does it render', () => {
    const props = {
      name: 'name',
    };
    const { container } = render(
      <DateAndTimePickerProviderWrapperTestUtil>
        <FormikWrapperTestUtil Component={TimeField} props={props} />
      </DateAndTimePickerProviderWrapperTestUtil>,
    );
    expect(container).toMatchSnapshot();
  });
});
