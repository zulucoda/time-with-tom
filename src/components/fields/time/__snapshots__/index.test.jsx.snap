// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Fields - Time - Unit Test does it render 1`] = `
<div>
  <form
    action="#"
  >
    <div
      class="sc-AxjAm jkaSBu"
    >
      <div
        class="MuiFormControl-root MuiTextField-root"
        format="HH:mm"
      >
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            name="name"
            placeholder="18:00"
            readonly=""
            type="text"
            value="02:00"
          />
        </div>
        
      </div>
    </div>
  </form>
</div>
`;
